src/cr/cube/matrix/measure.py

```diff
@@ -38,12 +39,34 @@ def column_comparable_counts(self):
 
     @lazyproperty
     def column_index(self):
-        """_ColumnIndex measure object for this cube-result."""
+        """_ColumnIndex measure object for this cube-result.
```

Comment:
There's no demand for smoothed column-index measure yet, let's not implement it until we need it. A lot could change between then and now and maybe we never need it (YAGNI).

Reply:
IMHO, considering that we supported col index smoothed before (this was for free cause in the old implementation col index was directly computed from the smoothed col proportions), I'm inclined to leave it as well after the consolidation because even if it's not exposed in the webapp or the exporter it (maybe) is used through the API or via 3rd party apps of customers. If we remove it we will introduce a regression and that's the reason I would leave it here.

Reply:
Yeah, I'm okay with that.